Formula/fourstore.rb

```diff
@@ -27,6 +27,7 @@ class Fourstore < Formula
   depends_on "pcre"
   depends_on "raptor"
   depends_on "rasqal"
+  depends_on "readline"
```

Comment:
The build fails on Linux because it also needs `dbus` as a dependency.

Reply:
Fixed